src/components/Asset/AssetActions/Consume.tsx

```diff
@@ -20,7 +20,7 @@ import { useIsMounted } from '@hooks/useIsMounted'
 import { usePool } from '@context/Pool'
 import { useMarketMetadata } from '@context/MarketMetadata'
 
-export default function Download({
+export default function Consume({
```

Comment:
don't change it :)

Reply:
I changed it because I found that in v4 we call this component's [context](https://github.com/oceanprotocol/market/tree/v4/src/components/Asset/AssetActions/index.tsx#L123:L123) as "Consume" instead of download. I just thought "Consume" makes more sense than download, but if not, we can just change all to "Download".

Reply:
That is just an old reference, it should have been Download there as well.